index.yaml

```diff
@@ -23,7 +23,7 @@ entries:
     version: 0.1.1
   - apiVersion: v2
     appVersion: v0.2.0
-    created: "2021-04-27T07:33:46.70437909Z"
+    created: "2022-01-20T20:01:59.52756959Z"
```

Comment:
Is it possible to not have the `created` dates all changed to current date? Release dates in history might be important for us and for plugin users.

Reply:
Yea, I can copy them over